src/plugins/intel_cpu/src/emitters/snippets/brgemm_base.hpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2020-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "emitters/snippets/cpu_kernel_executor_table.hpp"
+#include "emitters/utils.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "snippets/lowered/loop_info.hpp"
+#include "snippets/lowered/loop_manager.hpp"
+#include "utils/general_utils.h"
+
+namespace ov {
+namespace intel_cpu {
+
+struct BrgemmBaseKernelConfig : public snippets::KernelExecutorBase::GenericConfig {
+public:
+    BrgemmBaseKernelConfig() = default;
+
+    bool is_completed() const override;
+    bool is_empty() const;
+
+    void update(int64_t M, int64_t N, int64_t K, float beta);
```

Comment:
Looks like we don't use this updater. Also I think we should have only one full `update` method to avoid misunderstandings: the updater should update all fields of config.

Could you please leave only one `update` method with full semantic?

Reply:
with changes in previous comment, partial `update()` is removed. Thanks Alexandra!